frontend/src/App.tsx

```diff
@@ -91,9 +93,9 @@ function MobileAppToolbar() {
 function AppToolbar() {
   return (
     <Toolbar className={styles.AppToolbar}>
-      <LinkWithStickyParams to="/">
+      <Button component={Link} to="/">
```

Comment:
Why can't you use the ReactRouterLinkButton here too?

Reply:
updated to accept react nodes too